tests/integration-tests/tests/cloudwatch_logging/test_cloudwatch_logging.py

```diff
@@ -139,7 +139,7 @@ def _dump_cluster_log_state(self):
     @staticmethod
     def _base_os_to_platform(base_os):
         """Turn the name of a base OS into the platform."""
-        # Special case: alinux is how the config file refers to amazon linux, but in the chef cookbook
+        # Special case: alinux2 is how the config file refers to amazon linux 2, but in the chef cookbook
         # (and the cloudwatch log config produced by it) the platform is "amazon".
         translations = {"alinux": "amazon"}
```

Comment:
does this key need to be changed to `"alinux2"` as well?

Reply:
Good catch, fixed